django_ipam/base/admin.py

```diff
@@ -160,7 +160,7 @@ def response_add(self, request, *args, **kwargs):
         if request.POST.get('_popup'):
             return HttpResponse("""
                <script type='text/javascript'>
```

Comment:
```suggestion
            return HttpResponse(f"""
               <script type='text/javascript'>
                  opener.dismissAddAnotherPopup(window);
                  opener.dismissAddAnotherPopup(window,{request.POST.get('ip_address')});
               </script>
             """)
```

It looks ugly, can you please try using f-string here? :smile: